### DISCUSSION TITLE
w-auto

### DISCUSSION THREAD

**Discussion by JamesTheBald** [tailwindlabs/tailwindcss#6488] [Ideas]
Hi, I've come across a hole in Tailwind's documentation: a description for w-auto ("width : auto"). Or, at the least, I don't see any relevant results for w-auto using the Algolia-powered search on tailwind.com.
Cheers,
James the Bald

**Comment by NovusOpus** [comment]
That Utility/Class already exist and is working:  

Documentation Page: [Sizings > Width](https://tailwindcss.com/docs/width), scroll the class list to find `w-auto`

Example of  `w-auto`: [Tailwind Play](https://play.tailwindcss.com/dKELuSNdoa)

---

Update: Seems like you edited the Original Discussion.

The reason there's no example in the docs is because: `w-auto` or CSS equivalent: `width: auto`; is the default value, so you don't even have to use the class at all, unless you have some specific use cases.

**Reply by JamesTheBald** [reply]
Ah yes, there it is. Thanks. w-auto is after all the fixed widths, and before the fractional widths. 
So the only issue is that "w-auto" does not return any results on Tailwind.com's Algolia search.

**Reply by NovusOpus** [reply]
@JamesTheBald I may put a `PR` adding a reference for users looking for What does it do.

Update:
Added a [Pull requests](https://github.com/tailwindlabs/tailwindcss.com/pull/1041) for a Definition and Example.

**Reply by JamesTheBald** [reply]
Great, thanks!
